src/merge.js

```diff
@@ -243,8 +243,9 @@ class Merge {
         doc.creationDate = file.creationDate
         if (this.sameBinary(file, doc)) {
           if (doc.size == null) { doc.size = file.size }
-          if (doc.class == null) { doc.class = file.class }
-          if (doc.mime == null) { doc.mime = file.mime }
+
+          doc.class = file.class
+          doc.mime = file.mime
```

Comment:
The goal was to prevent changing the mimetype when binary content didn't change.